css/amp.css

```diff
@@ -168,10 +168,10 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
 }
 
 
-.i-amphtml-blur{
+.i-amphtml-blur {
   filter: blur(20px) !important;
   transform: scale(1.1) !important;
-  transition: opacity 1s ease !important;
+  transition: opacity 1s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
```

Comment:
Why `cubic-bezier` and not `ease`?

Reply:
I actually asked for it in the previous review, that's the material deceleration curve.
However, I believe the animation should not last more than 300ms